placement/types.go

```diff
@@ -36,8 +36,8 @@ type Algorithm interface {
 	// RemoveHost removes a host from the cluster
 	RemoveHost(p Snapshot, h Host) (Snapshot, error)
 
-	// ReplaceHost replace a host with a new host
-	ReplaceHost(p Snapshot, leavingHost, addingHost Host) (Snapshot, error)
+	// ReplaceHost replace a host with new hosts
+	ReplaceHost(p Snapshot, leavingHost Host, addingHost []Host) (Snapshot, error)
```

Comment:
This should probably be `addingHosts` for the parameter?  (just a method signature but probably worth updating)